developer/src/kmc/src/util/projectLoader.ts

```diff
@@ -4,7 +4,11 @@ import * as fs from 'fs';
 import { CompilerCallbacks, KeymanDeveloperProject, KeymanFileTypes, KPJFileReader } from "@keymanapp/common-types";
 import { InfrastructureMessages } from "../messages/messages.js";
 
-// TODO: merge with projectReader.ts, which should be renamed to projectRunner
+export const isProject = (filename: string): boolean =>
```

Comment:
Do you want the TODO of renaming projectReader.ts to projectRunner.ts?

Reply:
I'll go ahead and rename it on this PR, should have done that before...